src/background/index.ts

```diff
@@ -14,4 +14,14 @@ const store = (window.store = createStore())
 
 subscribe(store, browser, browser.tabs)
 
-chrome.tabs.onRemoved.addListener(tabId => store.dispatch({ type: 'TAB_CLOSED', payload: { tabId } }))
+chrome.windows.getAll({}, windows => store.dispatch({ type: 'chrome.windows.getAll', payload: { windows } }))
```

Comment:
The first parameter for getAll is optional. Do we need to add an empty object here?

Reply:
good call, pushed a fix